web/styles/popovers.css

```diff
@@ -478,10 +478,20 @@ ul {
     .user_profile_manage_own_edit_button,
     .user_profile_manage_others_edit_button {
         width: 25px;
+        height: 14px;
         text-align: center;
         font-size: 18px;
     }
 
+    #user_profile_manage_own_edit_link {
+        height: 25px;
+    }
```

Comment:
These CSS changes to the edit links should be in their own commit with a commit message explaining why these changes were made.

Reply:
[This commit](https://github.com/zulip/zulip/pull/26954/commits/d11f36cab983fb7394cc404d4b65dfd023738ca4) fixes CSS related issues of the alignment and color of edit icon in user profile.